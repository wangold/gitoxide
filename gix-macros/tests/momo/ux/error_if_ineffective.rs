/// If nothing is done, momo should fail.
#[gix_macros::momo]
fn main() {}
