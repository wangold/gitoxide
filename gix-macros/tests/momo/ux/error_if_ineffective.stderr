error: Couldn't apply a single conversion - momo is ineffective here
 --> tests/momo/ux/error_if_ineffective.rs:1:1
  |
1 | /// If nothing is done, momo should fail.
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/momo/ux/error_if_ineffective.rs:3:13
  |
3 | fn main() {}
  |             ^ consider adding a `main` function to `$DIR/tests/momo/ux/error_if_ineffective.rs`
